jenkins/edx-platform-test-notifier.py

```diff
@@ -0,0 +1,66 @@
+import click
+from github import Github
+import logging
+import os
+import sys
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def _get_github_token():
+    """
+    Get the github oauth token from
+    the environment variable GITHUB_TOKEN
+    """
+    return os.environ.get('GITHUB_TOKEN')
```

Comment:
I would suggest checking for null values here. os.environ.get won't throw an exception if the environment variable isn't set.